service/routes.py

```diff
@@ -17,14 +19,75 @@
 @app.route("/")
 def index():
     """ Root URL response """
+    res = {
```

Comment:
I remember prof mentioned that we should not use action words in the path, instead use GET/POST/DELETE to do the action
![image](https://github.com/CSCI-GA-2820-SU23-001/promotions/assets/113205868/6fcdd302-e414-4096-87c1-13c648bf0a65)